.github/workflows/build_wheels_macos.yml

```diff
@@ -80,6 +84,16 @@ jobs:
           else
             ${CONDA_RUN} bash "${PRE_SCRIPT}"
           fi
+      - name: Build clean
+        working-directory: ${{ inputs.repository }}
+        run: |
+          source "${BUILD_ENV_FILE}"
+          ${CONDA_RUN} python3 setup.py clean
+      - name: Uninstall MKL for TorchAudio x86 build
+        if: ${{ inputs.repository == 'pytorch/audio' && inputs.runner-type == 'macos-12' }}
+        run: |
+          source "${BUILD_ENV_FILE}"
+          conda uninstall -p "${CONDA_ENV}" -y mkl
```

Comment:
nit: Another option for us down the line is to make the setup-binary-builds action less prescriptive in terms of what it installs so we don't have to do things like this in the future.